iotdb-client/session/src/main/java/org/apache/iotdb/session/pool/SessionPool.java

```diff
@@ -404,6 +484,40 @@ private Session constructNewSession() {
               .thriftMaxFrameSize(thriftMaxFrameSize)
               .enableRedirection(enableRedirection)
               .version(version)
+              .useSSL(useSSL)
+              .trustStore(trustStore)
+              .trustStorePwd(trustStorePwd)
+              .build();
+    } else if (nodeUrls == null && !useSSL) {
+      session =
+          new Session.Builder()
+              .host(host)
+              .port(port)
+              .username(user)
+              .password(password)
+              .fetchSize(fetchSize)
+              .zoneId(zoneId)
+              .thriftDefaultBufferSize(thriftDefaultBufferSize)
+              .thriftMaxFrameSize(thriftMaxFrameSize)
+              .enableRedirection(enableRedirection)
+              .version(version)
+              .build();
+
+    } else if (nodeUrls != null && useSSL) {
```

Comment:
I feel like we can brute force the new parameters to be set in, without the useSSL judgment?